docs/source/api/types_trimmed.rst

```diff
@@ -41,26 +43,167 @@ Response Structure
 	:caption: Response Example
 
 	HTTP/1.1 200 OK
-	Connection: keep-alive
-	Access-Control-Allow-Credentials: true
-	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
-	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
-	Whole-Content-SHA512: Wh4z9VkNcOI8UzSTM77N+JFx5bP8yxRR4rg1fZIH40DI+0suOD36YhePUMMqMl6DIlIWjrnkj+iojuQ09oTzeg==
 	Cache-Control: no-cache, no-store, max-age=0, must-revalidate
-	Date: Wed, 12 Dec 2018 23:37:01 GMT
-	Access-Control-Allow-Origin: *
-	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
-	Content-Length: 1104
+	Content-Encoding: gzip
+	Content-Length: 389
 	Content-Type: application/json
+	Date: Fri, 31 Jan 2020 18:09:29 GMT
 	Server: Mojolicious (Perl)
+	Set-Cookie: mojolicious=...; expires=Fri, 31 Jan 2020 22:09:29 GMT; path=/; HttpOnly
+	Vary: Accept-Encoding
 
-	{ "response": [
+	{ "alerts": [
+		{
+			"level": "warning",
+			"text": "This endpoint is deprecated"
```

Comment:
should there be an alt pointing to `GET /types`

Reply:
... yeah. yeah there should.